src/autoscaler/scalingengine/cmd/scalingengine/main.go

```diff
@@ -79,25 +76,17 @@ func main() {
 	}
 	defer func() { _ = schedulerDB.Close() }()
 
-	httpStatusCollector := healthendpoint.NewHTTPStatusCollector("autoscaler", "scalingengine")
-	promRegistry := prometheus.NewRegistry()
-	healthendpoint.RegisterCollectors(promRegistry, []prometheus.Collector{
-		healthendpoint.NewDatabaseStatusCollector("autoscaler", "scalingengine", "policyDB", policyDb),
-		healthendpoint.NewDatabaseStatusCollector("autoscaler", "scalingengine", "scalingengineDB", scalingEngineDB),
-		healthendpoint.NewDatabaseStatusCollector("autoscaler", "scalingengine", "schedulerDB", schedulerDB),
-		httpStatusCollector,
-	}, true, logger.Session("scalingengine-prometheus"))
-
 	scalingEngine := scalingengine.NewScalingEngine(logger, cfClient, policyDb, scalingEngineDB, eClock, conf.DefaultCoolDownSecs, conf.LockSize)
 	synchronizer := schedule.NewActiveScheduleSychronizer(logger, schedulerDB, scalingEngineDB, scalingEngine)
 
-	httpServer, err := server.NewServer(logger.Session("http-server"), conf, scalingEngineDB, scalingEngine, synchronizer, httpStatusCollector)
+	server := server.NewServer(logger.Session("http-server"), conf, policyDb, scalingEngineDB, schedulerDB, scalingEngine, synchronizer)
+	httpServer, err := server.GetMtlsServer()
```

Comment:
The logs says that we are creating server. but the method is GetMtlsServer()... if we are really building something, then I would say "createMtlsServer" makes sense